tests/output/pjs/iframe.html

```diff
@@ -10,24 +10,23 @@
     <body>
         <div id="mocha"></div>
         <div id="output-area"></div>
-        <script src="../../../build/js/live-editor.core_deps.js"></script>
-        <script src="../../../build/js/live-editor.shared.js"></script>
-        <script src="../../../build/js/live-editor.output.js"></script>
+        <!-- Exposes 'i18n' as a global -->
+        <script src="../../../build/js/live-editor.i18n.js"></script>
+        <!-- Exposes Processing and JSHint as globals -->
         <script src="../../../build/js/live-editor.output_pjs_deps.js"></script>
-        <script src="../../../build/js/live-editor.output_pjs.js"></script>
+        <!-- Mocha, sinon, and test JS -->
         <script src="../../../node_modules/mocha/mocha.js"></script>
         <script src="../../../node_modules/expect.js/index.js"></script>
         <script src="../../../node_modules/sinon/pkg/sinon.js"></script>
         <script src="../../../testutil/json-reporter.js"></script>
-        <script src="test_utils.js"></script>
         <script src="../../shared_utils.js"></script>
         <script>
             runTestFiles([
-                "assert_test.js",
-                "ast_transform_test.js",
-                "async_test.js",
-                "jshint_test.js",
-                "output_test.js",
+                "../../../build/js/live-editor.tests_output_pjs_jshint.js",
```

Comment:
I'm guessing this change is because we can't run the source code inside phantomjs anymore, right?

Reply:
It was easier for each of the tests to be a built file, including all its dependencies. Otherwise I'd need to do a script include that exposed the relevant classes, right?
(Perhaps there's another way that I'm missing)

Reply:
The changes are fine.  I'm just trying to understand the rationale behind the changes. 🙂